README.rst

```diff
@@ -126,6 +126,12 @@ savings.
 
 ``$ export REMOTE_HREF=$(http :24817/pulp/api/v3/remotes/docker/docker/ | jq -r '.results[] | select(.name == "library/busybox") | ._href')``
 
+Whitelist tags when syncing content
+-----------------------------------
+
+Docker remotes can be configured to whitelist tags with the option ``whitelist_tags``.
```

Comment:
I'd prefer if this whole section was moved into the whitelist help_text. No need to do it now, I can grab it when I move to rst.